dll/win32/kernel32/client/sysinfo.c

```diff
@@ -74,6 +75,81 @@ GetSystemInfoInternal(IN PSYSTEM_BASIC_INFORMATION BasicInfo,
     }
 }
 
+UINT
+WINAPI
+GetRawSMBiosTableFromRegistry(OUT PVOID pData)
+{
+    LPWSTR RegistryKey = L"\\Registry\\Machine\\System\\CurrentControlSet\\Services\\mssmbios\\Data";
+    LPWSTR ValueNameStr = L"SMBiosData";
```

Comment:
static const LPCWSTR ... ?

Reply:
Why on earth would you make it `static`? o_O